tests/unit/utils_memory.cpp

```diff
@@ -194,134 +194,134 @@ TEST(MonotonicBufferResource, AllocationWithInitialBufferOnStack) {
-  CheckAllocation(&mem, max_block_size + 1, 1U);
-}
-
-// NOLINTNEXTLINE(hicpp-special-member-functions)
-TEST(PoolResource, BlockSizeIsNotMultipleOfAlignment) {
-  const size_t max_blocks_per_chunk = 3U;
-  const size_t max_block_size = 64U;
-  memgraph::utils::PoolResource mem(max_blocks_per_chunk, max_block_size);
-  EXPECT_THROW(mem.Allocate(64U, 24U), std::bad_alloc);
-  EXPECT_THROW(mem.Allocate(63U), std::bad_alloc);
-  EXPECT_THROW(mem.Allocate(max_block_size + 1, max_block_size), std::bad_alloc);
-}
-
-// NOLINTNEXTLINE(hicpp-special-member-functions)
-TEST(PoolResource, AllocationWithOverflow) {
-  {
-    const size_t max_blocks_per_chunk = 2U;
-    memgraph::utils::PoolResource mem(max_blocks_per_chunk, std::numeric_limits<size_t>::max());
-    EXPECT_THROW(mem.Allocate(std::numeric_limits<size_t>::max(), 1U), std::bad_alloc);
-    // Throws because initial chunk block is aligned to
-    // memgraph::utils::Ceil2(block_size), which wraps in this case.
-    EXPECT_THROW(mem.Allocate((std::numeric_limits<size_t>::max() - 1U) / max_blocks_per_chunk, 1U), std::bad_alloc);
-  }
-  {
-    const size_t max_blocks_per_chunk = memgraph::utils::impl::Pool::MaxBlocksInChunk();
-    memgraph::utils::PoolResource mem(max_blocks_per_chunk, std::numeric_limits<size_t>::max());
-    EXPECT_THROW(mem.Allocate(std::numeric_limits<size_t>::max(), 1U), std::bad_alloc);
-    // Throws because initial chunk block is aligned to
-    // memgraph::utils::Ceil2(block_size), which wraps in this case.
-    EXPECT_THROW(mem.Allocate((std::numeric_limits<size_t>::max() - 1U) / max_blocks_per_chunk, 1U), std::bad_alloc);
-  }
-}
-
-TEST(PoolResource, BlockDeallocation) {
-  TestMemory test_mem;
-  const size_t max_blocks_per_chunk = 2U;
-  const size_t max_block_size = 64U;
-  memgraph::utils::PoolResource mem(max_blocks_per_chunk, max_block_size, &test_mem);
-  auto *ptr = CheckAllocation(&mem, max_block_size);
-  test_mem.new_count_ = 0U;
-  // Do another allocation before deallocating `ptr`, so that we are sure that
-  // the chunk of 2 blocks is still alive and therefore `ptr` may be reused when
-  // it's deallocated. If we deallocate now, the implementation may choose to
-  // free the whole chunk, and we do not want that for the purposes of this
-  // test.
-  CheckAllocation(&mem, max_block_size);
-  EXPECT_EQ(test_mem.new_count_, 0U);
-  EXPECT_EQ(test_mem.delete_count_, 0U);
-  mem.Deallocate(ptr, max_block_size);
-  EXPECT_EQ(test_mem.delete_count_, 0U);
-  // CheckAllocation(&mem, max_block_size) will fail as PoolResource should
-  // reuse free blocks.
-  EXPECT_EQ(ptr, mem.Allocate(max_block_size));
-  EXPECT_EQ(test_mem.new_count_, 0U);
-}
+//
+//// NOLINTNEXTLINE(hicpp-special-member-functions)
+// TEST(PoolResource, SingleSmallBlockAllocations) {
+//   TestMemory test_mem;
+//   const size_t max_blocks_per_chunk = 3U;
+//   const size_t max_block_size = 64U;
+//   memgraph::utils::PoolResource mem(max_blocks_per_chunk, max_block_size, &test_mem);
```

Comment:
I would rather delete these tests as we are not using them anymore. If we will restore them, we will find your commit